content/tutorials/presence.textile

```diff
@@ -183,7 +183,7 @@ h2. Next steps
 <script src="https://apis.google.com/js/api.js"></script>
 <script type="text/javascript">
   window.addEventListener("load", function() {
-    var ably = new Ably.Realtime({ authUrl: 'https://www.ably.io/ably-auth/token/docs' }),
+    var ably = new Ably.Realtime({ authUrl: 'https://www.ably.io/ably-auth/token/docs', closeOnUnload: true }),
```

Comment:
Perhaps a comment here on why closeOnUnload is used?  I appreciate it's not public code, but devs will inspect